src/omnibase_core/mixins/mixin_discovery_responder.py

```
@@ -316,13 +332,15 @@ async def _send_discovery_response(
         self,
         original_event: OnexEvent,
         request: ModelDiscoveryRequestModelMetadata,
+        request_time: float,
```

Ruff (0.14.2)

Unused method argument: `request_time`

(ARG002)